### DISCUSSION TITLE
Discrepancy in VS Code Dark theme

### DISCUSSION THREAD

**Discussion by thdoan** [FIameCaster/prism-code-editor#15] [Q&A] [answered]
I just noticed the syntax highlighting on the demo page is different from v3.3.1 that I'm using. Steps to reproduce:

1. Go to https://prism-code-editor.netlify.app/
2. Change theme to VS Code Dark
3. Update Config tab: `language: 'javascript',`
4. Click Run
5. Paste this code:

```
function test() {
  const re = /^test$/i;
  const obj = {
    "str": "string",
    "int": 1,
    "bool": true
  };
}
```

You should see this:

![image](https://github.com/FIameCaster/prism-code-editor/assets/4110567/58f2b64f-169e-4411-ab06-9adff021b281)

Now go here:

https://stackblitz.com/edit/vitejs-vite-anepnm?file=src%2Fmain.ts

Notice the regex doesn't look exactly the same:

![image](https://github.com/FIameCaster/prism-code-editor/assets/4110567/76bd9dee-5198-41e4-9d47-67a0cab2c810)

Looks like it's missing some anchor tags that only exist on the demo page:

![image](https://github.com/FIameCaster/prism-code-editor/assets/4110567/c56a0d96-547d-4d61-b910-97059570a3ec)

**Comment by thdoan** [comment]
JSDoc highlighting is also different...

![image](https://github.com/FIameCaster/prism-code-editor/assets/4110567/e0407a0d-e40b-4998-954d-be66255586fa)

![image](https://github.com/FIameCaster/prism-code-editor/assets/4110567/167a85ca-29e8-4059-ba3f-1d9263227624)

**Reply by jonpyt** [reply, marked as answer]
Highlighting of regular expressions and JSDoc is done by optional dependencies. You need to import `prism-code-editor/prism/languages/regex` and `prism-code-editor/prism/languages/jsdoc` to enable these features.
